File: big_data_learning/spark-scala-java-learning/src/main/java/com/shawn/study/big/data/spark/scala/rdd/builder/CreateRddByLocalFile.scala
package com.shawn.study.big.data.spark.scala.rdd.builder

import org.apache.spark.SparkContext

object CreateRddByLocalFile {

  def main(args: Array[String]): Unit = {
    val sc = new SparkContext("local[*]", "create-rdd-by-local-file")
    val rdd = sc.textFile("data/simple-data/1.txt")
    rdd.collect().foreach(println)
    sc.stop()
  }

}

File: big_data_learning/spark-scala-java-learning/src/main/java/com/shawn/study/big/data/spark/java/core/rdd/builder/CreateRddByLocalFile.java
package com.shawn.study.big.data.spark.java.core.rdd.builder;

import org.apache.spark.SparkConf;
import org.apache.spark.api.java.JavaRDD;
import org.apache.spark.api.java.JavaSparkContext;

public class CreateRddByLocalFile {

  public static void main(String[] args) {
    SparkConf conf = new SparkConf().setAppName("create-rdd-by-local-file").setMaster("local[*]");
    JavaSparkContext context = new JavaSparkContext(conf);
    JavaRDD<String> rdd = context.textFile("data/simple-data/1.txt");
    rdd.collect().forEach(System.out::println);
    context.close();
  }
}
